### PR TITLE
Fix 'admin-passthru' command not supported error

### PR DESCRIPTION
**Pull request by xyan264**

### CODE REVIEW

File: plugins/microchip/switchtec-nvme-device.c [hunk 1]
```diff
@@ -199,7 +199,12 @@ int pax_nvme_ns_rescan(int fd)
 
 int pax_nvme_submit_passthru(int fd, int ioctl_cmd, struct nvme_passthru_cmd *cmd)
 {
-	fprintf(stderr, "%s not implemented.\n", __FUNCTION__);
+	if(ioctl_cmd == (int)NVME_IOCTL_ADMIN_CMD) {
```
Comment (kelvin-cao) [inline plugins/microchip/switchtec-nvme-device.c:202]: style nit: white space needed between if and (

File: plugins/microchip/switchtec-nvme-device.c [hunk 2]
```diff
@@ -199,7 +199,12 @@ int pax_nvme_ns_rescan(int fd)
...
+		return pax_nvme_submit_admin_passthru(fd, cmd);
+	} else {
+		fprintf(stderr, "%s only supports NVME_IOCTL_ADMIN_CMD.\n",
+			__FUNCTION__);
+	}
```
Comment (kelvin-cao) [inline plugins/microchip/switchtec-nvme-device.c:207]: Normally we do parameter check and error handling at the very beginning. Then the logic would be more clear. Like

``` 
        if (ioctl_cmd != (int)NVME_IOCTL_ADMIN_CMD)) {
                do error handling;
                return -1;
        }

        do normal processing;
        return 0;
```

**Review by kelvin-cao — APPROVED** [review]
Small changes needed, otherwise looks good.

**Comment by xyan264** [comment]
Code updated based on the above comments, merged to `devel`.

**Pull request merged**